swingset/src/main/java/com/nqadmin/swingset/SSBaseComboBox.java

```diff
@@ -264,16 +301,12 @@ protected BaseModel() {
 			BaseGlazedModel<M,O,O2> model = new BaseGlazedModel<>();
 			model.autoComplete = AutoCompleteSupport.install(_jc, model.getEventList(), null, model.getListItemFormat());
 			
-			model.autoComplete.setStrict(true);
 			model.autoComplete.setFilterMode(TextMatcherEditor.CONTAINS);
-			
+			model.autoComplete.setStrict(true);
```

Comment:
You placed model.autoComplete.setStrict(true); after setFilterMode() so going with your version.

Reply:
Didn't think so because it seemed fine in my testing, but since you've actually been inside GL was going with your version.